www/src/css/custom.css

```diff
@@ -291,3 +291,59 @@ html.v9 .trpcgif--v9 {
 .breadcrumbs__item {
   @apply flex items-center;
 }
+
+.header-social-link:hover {
+  opacity: 0.6;
```

Comment:
would add `@apply transition-colors opacity-60` here instead just because we have tailwind and transition hover is more pleasing.